dbtmetabase/_exposures.py

```diff
@@ -89,6 +91,13 @@ def extract_exposures(
                 uid=collection["id"],
                 models=("card", "dashboard"),
             ):
+                if (
+                    exclude_unverified_cards
+                    and item.get("moderated_status") != "verified"
+                    and item.get("model") == "card"
```

Comment:
```suggestion
                    item["model"] == "card"
                    and exclude_unverified_cards
                    and item.get("moderated_status") != "verified"
```

The `model` key must be there, otherwise we have bigger problems. And operators are lazy, so in order of specificity.

Reply:
Since this is a chain of `and` conditions, we should order the conditions from the most likely to be False to the least likely, right?

I figured the `exclude_unverified_cards` would by far be the least likely since it won't be used by most users. That's why I placed it first. I placed `and item.get("model") == "card"` last because I figured almost all items being iterated over are cards. Happy to defer to your expertise if you disagree though 👍

Reply:
Good point about `exclude`, in that case:

```suggestion
                    exclude_unverified_cards
                    and item["model"] == "card"
                    and item.get("moderated_status") != "verified"
```

Otherwise, you're checking a model-specific value before verifying that it's a model that can have that value.

Reply:
Very true!